### ISSUE TITLE
Allow meta tags as templateOptions

### ISSUE THREAD

**Issue by soosap** [revivek/oy#58]
Hi in my case I want to get rid of the annoying blue colored links set on Apple devices. In order to achieve that I would have to add the following meta tag to my email template:

**< meta name="format-detection" content="email=no" >**

So in general it would be nice if we could add meta tags just as we could add title.

**Comment by revivek** [comment]
Hi, thanks for the suggestion. I'm open to adding meta tags to the default template generator if they're general enough. I'd also like to minimize the number of configuration options in `renderTemplate`, for security and UX reasons. In this instance, the meta tag is too specific to add for everyone.

I'd suggest using `generateCustomTemplate`, which is the third (optional) parameter to `renderTemplate`. This allows you to provide a custom template, with your own meta tags and configuration. You can see how it's used here: https://github.com/revivek/oy/blob/ca50d8c31055a283fcd4442712ae38b736aad4b0/src/utils/Renderer.js There's an outstanding task to add documentation for this feature.

**Comment by yuindustries** [comment]
Hi @revivek,

I created a PR (https://github.com/revivek/oy/pull/68) to add some more details about how to implement custom templates. I was really struggling to figure out how to use the feature. So I wrote down some notes that would have been helpful to myself.

I really appreciate you creating and maintaining this project. I was feeling daunted by the task of generating a bunch of branded HTML emails, but overall Oy has made it much more pleasant!
